ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/PivotBlockRetrieverTest.java

```diff
@@ -269,6 +271,7 @@ public void shouldQueryBestPeersFirst(final DataStorageFormat storageFormat) {
 
   @ParameterizedTest
   @ArgumentsSource(PivotBlockRetrieverTestArguments.class)
+  @DisabledOnOs(OS.MAC)
```

Comment:
You can drop this once this pr merges: https://github.com/hyperledger/besu/pull/7253

Reply:
remved.